resources/scripts/components/dashboard/ServerRow.tsx

```diff
@@ -101,9 +97,9 @@ export default ({ server, className }: { server: Server; className?: string }) =
                         {server.allocations
                             .filter((alloc) => alloc.isDefault)
                             .map((allocation) => (
-                                <React.Fragment key={allocation.ip + allocation.port.toString()}>
+                                <Fragment key={allocation.ip + allocation.port.toString()}>
                                     {allocation.alias || ip(allocation.ip)}:{allocation.port}
-                                </React.Fragment>
+                                </Fragment>
                             ))}
                     </p>
                     {/* I don't think servers will ever have descriptions normall so I'll vaporize it */}
```

Comment:
Customers might have a server description especially in cases like people with server networks, a lot of networks use ID systems separate from the panel that might be used here.

Reply:
You're right, I can add it back. Noted (put on linear eventually) and will revisit